opencti-platform/opencti-front/src/private/components/data/Management.tsx

```diff
@@ -202,7 +203,7 @@ const Management = () => {
       <div style={{ paddingRight: isRightMenuManagementEnable ? '200px' : 0 }}>
         <Breadcrumbs elements={[
           { label: t_i18n('Data') },
-          { label: t_i18n('Management') },
+          { label: t_i18n('Restriction') },
```

Comment:
so we changed management to restriction ? is it confirmed with product @nino-filigran ?